src/core/api/index.js

```diff
@@ -1,4 +1,5 @@
 /* global fetch */
+/* eslint-disable arrow-body-style */
```

Comment:
I'm guessing this is for https://github.com/mozilla/addons-frontend/pull/1702/files#diff-9c6af2bc9310bfed1e429d02f827c559R65? If so, maybe this should just be a `disable-next-line`?

It's unrelated but I'd love to switch over to `arrow-body-style` always; I find it so much easier to scan.